jaxrs/src/main/java/org/jboss/as/jaxrs/JaxrsValidator.java

```diff
@@ -0,0 +1,39 @@
+package org.jboss.as.jaxrs;
+
+import java.util.regex.Pattern;
+
+import org.jboss.as.controller.OperationFailedException;
+import org.jboss.as.controller.operations.validation.ParameterValidator;
+import org.jboss.as.jaxrs.logging.JaxrsLogger;
+import org.jboss.dmr.ModelNode;
+
+public class JaxrsValidator implements ParameterValidator {
```

Comment:
I guess this class will not be needed once you do the LIST/OBJECT changes?

Reply:
Gone.